dev-perl/Statistics-Basic/Statistics-Basic-1.661.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=JETTERO
+DIST_VERSION=1.6611
+inherit perl-module
+
+DESCRIPTION="Statistics-Basic module"
+
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~sparc ~x86"
```

Comment:
You can't add other keywords to your **new package** than amd64 and x86, and even these require you to test it on these platforms. You'll have to request keywording through Bugzilla after the package has been merged.